documentation/ingles/aodvv2_process.md

```diff
@@ -0,0 +1,108 @@
+
+### Resource constraints
+ Nodes can have limited resources with respect to computational power, e.g., RAM and CPU power available, power supply, and cost of communication. In addition, there are constraints with regard to bandwidth on the wireless link, the effects of multiple access, fading, noise, and interference conditions may severely limit the transmission rate or even prevent link establishment.
+
+### Heterogeneity
+ Nodes can have varying characteristics with respect to the resource constraints speciﬁed above and be more or less willing to participate in routing.
+
+
+
+## AODVv2 Reactive Routing protocol
+
+This section provides a brief overview of AODV v2 protocol, in this one each node maintains a routing table (RT) containing information about the routes to be followed when sending messages to the others nodes of the network. The collective information in the nodes **"Routig table"** is at best a partial representation of network connectivity as it was at some times in the past.
+
+The basic routine of AODVv2 is similar to the one of AODV:  
+if a node **S** is required to send one or more data packets to a target node **T**, but does not have a route stored in  its routing table, it buffers the data and initiates a route discovery process by broadcasting or flooding a route request (RREQ) message. The RREQ is typically forwarded by intermediate nodes that  are not the target. An intermediate node that receives the RREQ message updates its routing table by creating entries for a route to the originator of the message **(node S)** and to all intermediate nodes that have forwarded the RREQ message (path accumulation). After that, the node typically re-broadcasts the request to its neighbours. This is repeated until the RREQ reaches the target node **T** that replies by unicasting a corresponding route reply (RREP) message back to the source **S** along the previous established path. To shorten route-discovery times, intermediate nodes are allowed to and should reply on behalf of the target if they know a route to **T**. An intermediate node that has this information updates its routing table as usual; after that it unicasts a RREP message back to the originator of the RREQ message **(nodeS)**, but it also unicasts a RREP message to the target **node T**. By this,routes between **S** and **T**, and between **T** and **S** are established. The RREP messages are forwarded by  the  intermediate nodes along previously established routes. When forwarding RREP messages, nodes create routing table entries for all nodes that have already forwarded that route, to the originator of the RREP message and, in case the originator is different to the target of the original RREQ, also to the target **node T**. After the RREP message has reached **node S**, a route from **S** to **T** is established and data packets can start to flow.
```

Comment:
```suggestion
The basic routine of AODVv2 is similar to the one of AODV:

if a node **S** is required to send one or more data packets to a target node **T**, but does not have a route stored in  its routing table, it buffers the data and initiates a route discovery process by broadcasting or flooding a route request (RREQ) message.

The RREQ is typically forwarded by intermediate nodes that  are not the target. An intermediate node that receives the RREQ message updates its routing table by creating entries for a route to the originator of the message **(node S)** and to all intermediate nodes that have forwarded the RREQ message (path accumulation). After that, the node typically re-broadcasts the request to its neighbours.

This process is repeated until the RREQ reaches the target node **T** that replies by unicasting a corresponding route reply (RREP) message back to the source **S** along the previous established path.

To shorten route-discovery times, intermediate nodes are allowed to and should reply on behalf of the target if they know a route to **T**. An intermediate node that has this information updates its routing table as usual; after that it unicasts a RREP message back to the originator of the RREQ message **(node S)**, but it also sends a unicast RREP message to the target **node T**. By this, routes between **S** and **T**, and between **T** and **S** are established.

The RREP messages are forwarded by  the  intermediate nodes along previously established routes. When forwarding RREP messages, nodes create routing table entries for all nodes that have already forwarded that route, to the originator of the RREP message and, in case the originator is different to the target of the original RREQ, also to the target **node T**.

After the RREP message has reached **node S**, a route from **S** to **T** is established and data packets can start to flow.
```

I've been splitted this paragraph for readability, it's well explained, I just added a few new lines where it makes sense. Overall that's a good explanation.

You can add this change by committing this suggestion on GitHub or applying it locally, the way you prefer :wink: